packages/docs/PropTables/MarginPropTable.tsx

```diff
@@ -5,37 +5,37 @@ import { NextLink, Prop, PropTable, PropTableWrapper } from '../components';
 const marginProps: Prop[] = [
   {
     name: 'margin',
-    types: <NextLink href="/margin">Margin</NextLink>,
+    types: <NextLink href="/spacing">Spacing</NextLink>,
```

Comment:
Updated these as `Spacing` since I think it's more descriptive. Is that ok? Thoughts?